README.md

```diff
@@ -61,7 +61,7 @@ You'll need:
 To install `rover`:
 
 ```sh
-curl -sSL https://rover.apollo.dev/nix/v0.4.8 | sh
+curl -sSL https://rover.apollo.dev/nix/v0.5.0-rc.0 | sh
```

Comment:
should this be rc.1?

Reply:
Great catch! Fixed in https://github.com/apollographql/supergraph-demo-fed2/pull/106/commits/7bd5c47fac02081288f44305e24cedb634a92da9